### PR TITLE
Fix typo and grammar mistakes

### PR DESCRIPTION
**Pull request by dnnr**
All pull requests must have an associated issue in the issue tracker. If there
isn't one, please go open an issue describing the defect, deficiency or desired
feature. You can read more about our issue and PR processes in the
[wiki](https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management).

- [ ] issues: fixes #xxxx
- [ ] tests added / passed
- [ ] release document entry (if new feature or API change)

### CODE REVIEW

**Comment by dnnr** [comment]
Not sure if there's any point in opening an issue for that, as the PR message template suggests...

File: bokeh/models/tools.py [hunk 1]
```diff
@@ -994,7 +994,7 @@ class HoverTool(Inspection):
     have a matching value for their ``name`` attribute will be used.
     """)
 
-    renderers = Either(Auto, List(Instance(Renderer)), defatult="auto", help="""
+    renderers = Either(Auto, List(Instance(Renderer)), default="auto", help="""
```
Comment (mattpap) [inline bokeh/models/tools.py]: Keyword arguments strike again. It's interesting that this wasn't picked up by any tests.

Comment (mattpap) [reply bokeh/models/tools.py]: Actually `"auto"` is the intrinsic default value and specifying `default` manually is just either pedantry or is supposed to aid documentation.

File: bokeh/models/annotations.py [hunk 2]
```diff
@@ -1002,7 +1002,7 @@ class Tooltip(Annotation):
     """)
 
     show_arrow = Bool(default=True, help="""
-    Whether tooltip's arrow should be showed.
+    Whether tooltip's arrow should be shown.
```
Comment (mattpap) [inline bokeh/models/annotations.py]: As far I can tell (not a native speaker), both are correct, just, I suppose, "shown" is the preferred one.

**Comment by mattpap** [comment]
> Not sure if there's any point in opening an issue for that, as the PR message template suggests...

It's advised, but we usually skip this in such trivial cases.

**Comment by mattpap** [comment]
Test failures are unrelated and will be fixed in PR #8406.

**Pull request merged**